### PR TITLE
New NCN Overview Page

### PR DESCRIPTION
**Pull request by rustydb**
# Description

Add a new `README.md` page detailing all the kernel parameters.

Lint several of the background pages.

Include information about the Python installations in the overview page.

# Checklist Before Merging

- [ ] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [ ] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [x] My commits or Pull-Request Title contain my JIRA information, or I don't have a JIRA.

[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

File: background/ncn_images.md [hunk 1]
```diff
@@ -1,176 +1,157 @@
 # NCN Images
 
 The management nodes boot from NCN images which are created from layers on top of a common base image.
-The common image is customized with a Kubernetes layer for the master nodes and worker nodes.
-The common image is customized with a storage-ceph layer for the utility storage nodes..
+The common image is customized with a `kubernetes` layer for the master nodes and worker nodes.
+The common image is customized with a `storage-ceph` layer for the utility storage nodes..
```
Comment (nrockershousen) [inline background/ncn_images.md:5]:
```suggestion
The common image is customized with a `storage-ceph` layer for the utility storage nodes.
```

File: background/ncn_images.md [hunk 2]
```diff
@@ -1,176 +1,157 @@
...
-#### Overview of NCN Images
+### Overview of NCN Images
 
 There are several flavors of NCN images, each share a common base image. When booting NCNs an admin or user will need to choose between
 stable (Release) and unstable (pre-release/dev) images.
 
 > For details on how these images behave and inherit from the base and common images, see [node-image-docs][1].
 
-In short, each image (i.e. Kubernetes and storage-ceph) inherit from the non-compute-common layer. Operationally these are all
+In short, each image (i.e. `kubernetes` and `storage-ceph`) inherit from the non-compute-common layer. Operationally these are all
```
Comment (nrockershousen) [inline background/ncn_images.md:21]:
```suggestion
In short, each image (for instance, `kubernetes` and `storage-ceph`) inherit from the non-compute-common layer. Operationally, these are all
```

File: background/ncn_images.md [hunk 3]
```diff
@@ -1,176 +1,157 @@
...
 that matter; the common layer, Kubernetes layer, Ceph layer, and any other new images.
 
-To boot an NCN, you need 3 artifacts for each node-type (kubernetes-manager/worker, ceph):
+To boot an NCN, you need 3 artifacts for each node-type (`kubernetes-master/worker`, `storage-ceph`):
```
Comment (nrockershousen) [inline background/ncn_images.md:24]:
```suggestion
To boot an NCN, you need three artifacts for each node-type (`kubernetes-master/worker`, `storage-ceph`):
```

File: background/ncn_images.md [hunk 4]
```diff
@@ -1,176 +1,157 @@
...
+   Waiting on 9 to request DHCP ...
+   Waiting on 9 to request DHCP ...
+   All [9] expected BMCs have requested DHCP.
+   /root/bin/set-sqfs-links.sh is creating boot directories for each NCN with a BMC that has a lease in /var/lib/misc/dnsmasq.leases
+   NOTE: Nodes without boot directories will still boot the non-destructive iPXE binary for bare-metal discovery usage.
+   WARNING: CSM_RELEASE was not set, images will be stored in their default location on the node(s) at /run/initramfs/live/LiveOS/
+   /var/www is ready.
    ```
 
 1. (`pit#`) View the currently set links
```
Comment (nrockershousen) [inline background/ncn_images.md:101]:
```suggestion
1. (`pit#`) View the currently set links.
```

File: background/ncn_networking.md [hunk 5]
```diff
@@ -34,33 +34,34 @@ The following table includes information about the different NCN network interfa
...
-   * `sun`: internal/storage network connection
-   * `hsn`: high-speed connection
-   * `lan`: external/site-connection
+* `mgmt`: internal/management network connection
+* `sun`: internal/storage network connection
+* `hsn`: high-speed connection
+* `lan`: external/site-connection
 
-The source code for the rule generation is in [metal-ipxe][1], but for technical information on the PCI configuration/reading please read on.
+The source code for the rule generation is in [`metal-ipxe`][1], but for technical information on the PCI configuration/reading please read on.
```
Comment (nrockershousen) [inline background/ncn_networking.md:55]:
```suggestion
The source code for the rule generation is in [`metal-ipxe`][1]. Continue reading for technical information on the PCI configuration/reading.
```

File: background/ncn_plan_of_record.md [hunk 6]
```diff
@@ -8,70 +8,71 @@ This document outlines the hardware necessary to meet CSM's Plan of Record (PoR)
...
+* *CEPH:* `8x` SSDs of any size
 
 > **`NOTE:`** Any available disk that is not consumed by the operating system will be used for CEPH, but a node needs a minimum of 8 disks for making an ideal CEPH pool for CSM.
 
 #### Storage NICs
 
 > **`NOTE:`** The 2nd port on each card is filled but not configured (reserved for future use).
 
-- _Management Network:_ 2x PCIe cards, each with 2 heads/ports for a total of 4 ports split between two PCIe cards
+* *Management Network:* `2x` PCIe cards, each with 2 heads/ports for a total of 4 ports split between two PCIe cards
```
Comment (nrockershousen) [inline background/ncn_plan_of_record.md:78]:
```suggestion
* *Management Network:* `2x` PCIe cards, each with two heads/ports for a total of four ports split between two PCIe cards
```

**Review by nrockershousen — APPROVED** [review]

**Review by heemstra — APPROVED** [review]
